integration_test/robots/send_friend_request_robot.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quellenreiter_app/widgets/opponent_card.dart';
+
+import '../test_tools.dart';
+
+class SendFriendRequestRobot {
```

Comment:
Instead of constructing the bot with the `_friendName`, we could make it reusable and just pass the friendName in `searchForFriend()`. We would at least save lots of instantiations in the future? (beware of YAGNI, tho. Dont know if this is an unnecessary optimization)